src/DashTextareaAutocomplete.jl

```diff
@@ -16,14 +17,14 @@ function __init__()
             [
                 DashBase.Resource(
     relative_package_path = "dash_textarea_autocomplete.min.js",
-    external_url = "https://unpkg.com/dash_textarea_autocomplete@1.1.0/dash_textarea_autocomplete/dash_textarea_autocomplete.min.js",
+    external_url = "https://unpkg.com/dash-textarea-autocomplete@1.1.0/dash_textarea_autocomplete/dash_textarea_autocomplete.min.js",
```

Comment:
Do we actually need these `externarl_url` definitions now that we're set to use Artifacts and a tarball?

Reply:
Well, we didn't need them before since they never actually worked 😏 

If we're going to have an external_url resource, it'd be nice if it was a functioning link even if we won't rely on it because of artifacts.

Maybe there's a case where artifacts can't be acquired and this would be the fallback ?